### DISCUSSION TITLE
2007/12/17/%E3%80%8A%E8%8E%BA%E8%8E%BA%E4%BC%A0%E3%80%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#195] [Announcements]
# 2007/12/17/%E3%80%8A%E8%8E%BA%E8%8E%BA%E4%BC%A0%E3%80%8B

最近在《百家讲坛》中听到《莺莺传》，于是查阅了更多资料。《莺莺传》是唐代元稹所著的一部短篇传奇小说，记载的是一件真人真事。后来被元代王实甫改编成脍炙人口的《西厢记》，在中国文学史上留下了浓墨重彩的一笔。

https://blog.qizhen.xyz/2007/12/17/%E3%80%8A%E8%8E%BA%E8%8E%BA%E4%BC%A0%E3%80%8B

**Comment by ruanqizhen** [comment]
之前博客下的留言：

GuangGuang
2009年12月11日 11:51 编辑
从元稹后来的为文来看，他还算是一个有勇气，可以称之为男人的人：因为他敢于自我批评，敢于剖析自我。现代社会又有多少满嘴仁义道德，一肚子男盗女娼的人？特别是那些所谓的官们！！

回复
Qizhen
2008年03月6日 17:39 编辑
贴一首元稹回忆莺莺的诗：“寒轻夜浅绕回廊，不辨花丛暗辨香。忆得双文胧月下，小楼前后捉迷藏。”。读其诗，我可以感同身受与心爱姑娘一起游戏时的快乐。但想想，这诗意的源头来自一个伪君子的笔下，谁知他的感情有多少是真，有多少是假呢？

回复
幽灵公主
2008年01月8日 17:38 编辑
哎  一声叹息

回复
竹贤
2007年12月28日 21:45 编辑
任何时候，我们都必须遵守游戏规则。规则有两种：硬性的规则是法律法规；软性的规则则是社会公认的道德和文化传统。
只要不违背游戏规则，你是自由的。如果违背了游戏规则，那带来的就是…..
 

回复
传杰
2007年12月18日 09:19 编辑
元稹的选择绝对是世人普遍的选择。换做现在，我想，绝大部分人也会如此选择。何况，莺莺还是个青楼女子，当然并不是说青楼女子就天生下贱，而是她们难以获得认同。试想，某职业经理男，面对一个选择，一边是某上市公司老总的招婿，一边是一个洗头房的姑娘，要二选其一。他会如何选择？答案很明显。
张生并不龌龊，他就是我们身边的某人，有时候，甚至就是我们自己。
